### ISSUE TITLE
Add Webi to list of install methods again.

### ISSUE THREAD

**Issue by coolaj86** [ajeetdsouza/zoxide#637]
Hey @ajeetdsouza,

Would you accept a PR to add Webi back to the install methods?

I believe that over the years we've been able to find solutions for all of the most important philosophical issues and edge case bugs that affected the installer.

If there's anything outstanding, I'd love to hear about it.

**Comment by ajeetdsouza** [comment]
Hey @coolaj86, I appreciate the work you've done on `webi`. My initial motivation for switching over to Webi was to ease maintenance, but it ended up doing the opposite:

- Issues that were reported now took much longer to resolve, because they now relied on me getting time to figure out the issue with `webi`, create a PR, and it getting reviewed / merged there.
- There were some philosophical issues I had reported right at the start (`webi` modifying config files, installing itself onto your computer without warning, etc.). I had assumed that this would not trouble too many people, but quite a few users expressed discontent with these changes. I'm not sure if this has been fixed yet.
- `webi`, being an actively developed repository, had bugs that broke the installer a couple of times. While this is inevitable, it does mean that it involves a time investment from me every time it breaks - OTOH, the current zoxide installer has not changed since it was (re) added.

I may reconsider this in future, but for now, I'd prefer to stick with the current installation script. Hope you understand!
